conf/config.yaml

```diff
@@ -87,7 +87,7 @@ data:
               password: "123456"
               database: employees_0003
               weight: r10w10
-        - name: employees_shadow
+        - name: employees_show
```

Comment:
why fix this name?

Reply:
keep group name same with db.

Reply:
use employees_shadow replace all employees_show